src/app/content/highlights/components/cardUtils.ts

```diff
@@ -11,28 +11,26 @@ import { HighlightData } from '../types';
 
 export const getHighlightOffset = (
   container: HTMLElement | undefined,
-  highlight: Highlight,
-  expandableAncestorsSelector?: string
+  highlight: Highlight
 ) => {
   if (!container || !highlight.range || !highlight.range.getBoundingClientRect) {
     return;
   }
 
   const {top, bottom } = highlight.range.getBoundingClientRect();
 
-  if (expandableAncestorsSelector) {
-    const highlightElement = highlight.elements[0] as HTMLElement;
-    const expandableAncestor = highlightElement && highlightElement.closest(expandableAncestorsSelector);
-
-    if (expandableAncestor
-      && expandableAncestor.getBoundingClientRect().bottom
-      < highlightElement.getBoundingClientRect().top
-    ) {
-      return {
-        bottom: -9999,
-        top: -9999,
-      };
-    }
+  const highlightElement = highlight.elements[0] as HTMLElement;
+  const expandableAncestor = highlightElement
+    && highlightElement.closest('[data-type="solution"][aria-expanded="false"]');
+
+  if (expandableAncestor
+    && expandableAncestor.getBoundingClientRect().bottom
+    < highlightElement.getBoundingClientRect().top
+  ) {
+    return {
+      bottom: -9999,
```

Comment:
this value could be a constants since it is used in `updateStackedCardsPositions`